Resources/Prototypes/_StarLight/Roles/play_time_trackers.yml

```diff
@@ -19,6 +19,9 @@
 - type: playTimeTracker
   id: JobBlueShield
 
+- type: playTimeTracker
+  id: ConsortiumBlueShield
```

Comment:
follow the job naming scheme of Job{JOBNAME}. make sure to update all references to this tracker aswell